mockdate/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for mockdate 2.0
+// Project: https://github.com/boblauer/MockDate
+// Definitions by: Bruno Leonardo Michels <https://github.com/brunolm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "mockdate" {
+    export default MockDate;
```

Comment:
This package does not define a default export. (See [our README](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default), [their README](https://github.com/boblauer/MockDate#usage)).
Instead, just use:
```ts
export as namespace MockDate;
export function set(date: { valueOf(): number } | number | string, timezoneOffset?: number): void;
export function reset(): void;
```